docs/src/install/index.md

```diff
@@ -221,3 +221,42 @@ The packages can be viewed directly from the [Cloudsmith.io](https://cloudsmith.
     </td>
 </tr>
 </table>
+
+## Shell completions
+To make using the `tedge` CLI easier, as of TODO VERSION, `tedge` provides
```

Comment:
@reubenmiller this obviously needs populating with a version number at some point. Do we just assume it's going to be `1.5.0` or do we have a better way to handle it?

Reply:
I would add a `::note` section saying that tab completion is only available in >= 1.5.0.

Reply:
Done